test/proposal_store_test.rb

```diff
@@ -0,0 +1,44 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2015 SUSE GmbH, All Rights Reserved.
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact SUSE GmbH.
+#
+# To contact SUSE about this file by physical or electronic mail, you may find
+# current contact information at www.suse.com.
+# ------------------------------------------------------------------------------
+
+require_relative "test_helper"
+
+require "migration/proposal_store"
+
+describe Migration::ProposalStore do
+  ps = Migration::ProposalStore.new(nil)
+  
+  describe ".headline" do
+    it "returns a headline" do
+      expect(ps.headline).to eq "Migration proposal"
+    end
+  end
+
+  describe ".proposal_names" do
+    it "returns an update proposal" do
+      expect(ps.proposal_names).to eq ["update_proposal"]
+    end
+  end
+
+  describe ".presentation_order" do
+    it "returns a presentation order" do
+      expect(ps.presentation_order).to eq ["update_proposal"]
+    end
+  end
```

Comment:
well, private method is not tested here...it means there should be also tests for public methods from `::Installation::ProposalStore` that changed due to different private method implementation.